assignments/function-conversion.js

```diff
@@ -1,23 +1,19 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {};
+let myFunction = () => {};
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
+let anotherFunction = (param1) => {return param1;};
 
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
+let add = (param1, param2) => {return param1 + param2;};
+add(1,2);
 
 let subtract = function (param1, param2) {
   return param1 - param2;
 };
-subtract(1,2); //?
+subtract(1,2); 
```

Comment:
The return statements are an extra step here - you only need to tell the function what to do, then when you call the function (`subtract(1,2);`) - the return is implicit there.